cypress/support/fragments/inventory/holdingsMove/inventoryInstancesMovement.js

```diff
@@ -12,6 +12,11 @@ export default {
     cy.do(Modal('Confirm move').find(Button('Continue')).click());
     cy.expect(HTML({ id:'inventory-module-display' }).exists());
   },
+  moveFromMultiple(holdingName) {
+    cy.do(Accordion({ label: including(`Holdings: ${holdingName}`) }).find(Button('Move to')).click());
+    cy.do(DropdownMenu().find(Button()).click());
+    cy.do(Modal('Confirm move').find(Button('Continue')).click());
```

Comment:
Modal('Confirm move').find(Button('Continue') move this button to const

Reply:
done